lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml

```diff
@@ -477,6 +448,9 @@ states:
     events:
       next:
         targetState: CRI_HMRC_KBV_J6
+        checkIfDisabled:
```

Comment:
Don't think we need this here as it's covered by the route to this state itself?